source/usb/usb_lib.c

```diff
@@ -2037,6 +2175,16 @@ const U8 USBD_ConfigDescriptor[] = {
     MSC_EP
 #endif
 
+#if (USBD_CDC_ACM_ENABLE)
+#if (USBD_MULTI_IF)
+    CDC_ACM_DESC_IAD(USBD_CDC_ACM_CIF_NUM, 2)
+#endif
+    CDC_ACM_DESC_IF0
+    CDC_ACM_EP_IF0
+    CDC_ACM_DESC_IF1
+    CDC_ACM_EP_IF1
+#endif
```

Comment:
Why was this code moved? If this didn't fix a problem it may be good to move it back to reduce the diff. If it did fix a bug you'll probably want to make the same change for ```USBD_ConfigDescriptor_HS``` later on in this file. @devanlai do you have any insights on this?

Reply:
I'm out of time to review it further this morning, but Windows is stricter about devices that report a USB 2.0 (as is required for the WebUSB Binary Object Store descriptor). If the interface numbers don't match the order in which the interfaces appear in the configuration descriptor, Windows will report an error. This may be related to the HS interface not functioning.